docs/DeveloperGuide.adoc

```diff
@@ -98,18 +104,18 @@ image::LogicClassDiagram.png[]
 *API* :
```

Comment:
CommandResult and StatsPayload have dual dependency?

Reply:
No, only one way. Seems to have overlook the line overlapping the other dependency arrows.
Thanks for the catch!